Simcoe/Simcoe.swift

```diff
@@ -83,8 +83,10 @@ public final class Simcoe {
         tracker.track(event: event)
     }
 
-    fileprivate func findProviders<T>() -> [T] {
-        return providers
+    fileprivate func find<T>(_ providers: [AnalyticsTracking]?) -> [T] {
+        let activeProviders = providers != nil ? providers! : self.providers
```

Comment:
You can use the nil coalescing operator ?? to make this a bit more readable, ex:
`let activeProviders = providers ?? self.providers`